app/views/declaration/who_is_registering_this_registration.scala.html

```diff
@@ -21,6 +21,7 @@
 @(f: forms.Form2[_], model : Seq[ResponsiblePerson])(implicit lang:Lang,  m: Messages, request: Request[_])
 
 @header = {
+
```

Comment:
any reason why here's no back link?

Reply:
Added alongside spec file for that view. My bad